RecoParticleFlow/PFProducer/plugins/PFProducer.cc

```diff
@@ -448,5 +455,10 @@ void PFProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptions)
   desc.add<std::vector<double>>("calibHF_b_HADonly", {1., 1., 1., 1., 1., 1., 1., 1., 1., 1.});
   desc.add<std::vector<double>>("calibHF_b_EMHAD", {1., 1., 1., 1., 1., 1., 1., 1., 1., 1.});
 
+  // resolution parameters for HF: EPJC 53(2008)139, doi:10.1140/epjc/s10052-007-0459-4
+  desc.add<double>("resolHF_a", 2.799)->setComment("HF resolution - stochastic term");
```

Comment:
as another future development item, we should consider how these values change with aging